tiptapy/__init__.py

```diff
@@ -144,6 +147,30 @@ def inner_render(self, node) -> str:
         return f'<div class="embed-wrapper {provider_name.lower()}-wrapper"><figure>{html}</figure></div>'  # noqa: E501
 
 
+class CodeBlock(BaseNode):
+    type = "code_block"
+    wrap_tag = "pre"
+
+    def is_renderable(self, node):
+        renderable = False
+        content = node.get("content", [])
+        if content:
+            text = content[0]
+            renderable = bool(text.get("text", ""))
+        return renderable
+
+    def inner_render(self, node):
+        attrs = node.get("attrs", {})
+        language = attrs.get("language", "")
+        content = node.get("content", {})[0]
+        text = content.get("text", "")
+        if language:
+            lexer = get_lexer_by_name(language, stripall=True)
+            formatter = HtmlFormatter()
+            html = highlight(text, lexer, formatter)
+        return f'<code>{html}</code>'
```

Comment:
Can we remove the extra <pre><code> at the start?, it adds extra margin in the rendered article.

Reply:
Removed `<pre><code>`